### PR TITLE
Cori noshifter

### PR DESCRIPTION
**Pull request by fredericpoitevin**

### CODE REVIEW

File: NoDocker/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+conda env list
+```
+
+### 4. install crystallography libraries
+
+Work in progres...
+
+## Usage
+```bash
+source /.../sfxPhasing/Docker/conda.local/env.local
```
Comment (chuckie82) [inline NoDocker/README.md]: replace ... with full path

File: NoDocker/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### 3. activate the sfx conda environment
+```bash
+source activate sfx
+conda env list
+```
+
+### 4. install crystallography libraries
+
+Work in progres...
```
Comment (chuckie82) [inline NoDocker/README.md]: progress

File: NoDocker/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Usage
+```bash
+source /.../sfxPhasing/Docker/conda.local/env.local
+source activate sfx
+source /.../ccp4-7.1/bin/ccp4.setup-sh
```
Comment (chuckie82) [inline NoDocker/README.md]: same here

File: NoDocker/README.md (hunk 2 above)
Comment (fredericpoitevin) [reply NoDocker/README.md]: this now points to https://github.com/slaclab/autosfx-dependencies

File: NoDocker/README.md (hunk 1 above)
Comment (fredericpoitevin) [reply NoDocker/README.md]: thanks, I've added more details on this. It's a bit verbose unfortunately, not sure what's the best way to point to these directories...

File: NoDocker/README.md (hunk 3 above)
Comment (fredericpoitevin) [reply NoDocker/README.md]: Thanks, see above. I added phenix too.

**Pull request merged**